legal/satzung.md

```diff
@@ -0,0 +1,122 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software unter OSI-anerkannten Open-Source-Lizenzen,
+    2. Unterstützung bei der Weiterentwicklung ebendieser Software, und
+    3. $(ZWECK)
```

Comment:
Schulungen, Dokumentation, Wartung, Support bei Veröffentlichungen, etc.

Reply:
Maybe we need to go a step back here, and first agree on the main aim "Förderung von Wissenschaft und Forschung". This is the results of the fact that we cannot write "Förderung von open-source Software" or something similar, right? And probably most of the oemof users work in some science related fields, so this is close ...

But I think we should at least think of and discuss some general alternatives. From my perspective, oemof supports the energy transition by providing (transparent) tools to support analysis and decision making within the Energiewende. Of course science is more neutral - but what is everybody writing at the beginning of his papers ;) something like Klimawandel, Energieziele, and so on ... and does e.g. the Zweck "Förderung einer nachhaltigen Energieversorgung" include the support for science?! – so far, my personal motivation. And I am wondering if we exclude something if we write “Förderung von Wissenschat und Forschung”. What do you think?

Reply:
There is an [exhaustive list](https://www.gesetze-im-internet.de/ao_1977/__52.html) what may be named as the primary goal. The item on the list that is closest to your considerations would be environment protection ("Förderung des Umweltschutzes"). And compared to that, science and research fit much better - at least in my opinion.

Reply:
Okay, I am fine with "Förderung von Wissenschaft und Forschung". So I will resolve this conversation.